Models/appointmentModel.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('../DataBase/connection');
+
+const AppointmentSchema = new mongoose.Schema({
```

Comment:
if we dont define const Schema = ...., then no need to have the word "new" before mongoose.Schema?

Reply:
I was wrong about that. Did a little more reading up on Schemas and I think we do need to create a new instance. Defining const Schema... simply lets us instantiate `new Schema` instead of `new mongoose.Schema`.